solr/solrj/src/test/org/apache/solr/client/solrj/impl/LBHttpSolrClientTest.java

```diff
@@ -16,33 +16,36 @@
  */
 package org.apache.solr.client.solrj.impl;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.io.IOException;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.solr.client.solrj.ResponseParser;
 import org.apache.solr.common.params.ModifiableSolrParams;
 import org.junit.Test;
 
-import java.io.IOException;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-
-/**
- * Test the LBHttpSolrClient.
- */
+/** Test the LBHttpSolrClient. */
 public class LBHttpSolrClientTest {
-  
+
   /**
    * Test method for {@link LBHttpSolrClient.Builder}.
-   * 
-   * Validate that the parser passed in is used in the <code>HttpSolrClient</code> instances created.
+   *
+   * <p>Validate that the parser passed in is used in the <code>HttpSolrClient</code> instances
+   * created.
    */
   @Test
-  // commented out on: 17-Feb-2019   @LuceneTestCase.BadApple(bugUrl="https://issues.apache.org/jira/browse/SOLR-12028") // added 20-Sep-2018
+  // commented out on: 17-Feb-2019
+  // @LuceneTestCase.BadApple(bugUrl="https://issues.apache.org/jira/browse/SOLR-12028") // added
```

Comment:
I've tried to remove all the ones I came across (since they were 2-3 years old at this point). `./src/test/org/apache/solr/client/solrj/io/stream` is specifically excluded in this PR - so not surprised they are there.